ztest/ztest.go

```diff
@@ -167,10 +185,15 @@ func FromYAMLFile(filename string) (*ZTest, error) {
 // auto" and maybe be gzip-compressed.  outputFormat may be any string accepted
 // by "zq -f".  If zq is empty, the query runs in the current process.  If zq is
 // not empty, it specifies a zq executable that will be used to run the query.
-func run(zq, ZQL, input, outputFormat string) (string, error) {
+func run(zq, ZQL, outputFormat string, inputs ...string) (string, error) {
```

Comment:
Please adjust the docstring to account for `input` -> `inputs` transition.

Reply:
ok